integration-tests/maven/src/test/java/io/quarkus/maven/it/CreateProjectMojoIT.java

```diff
@@ -111,40 +111,13 @@ public void testProjectGenerationFromScratch() throws MavenInvocationException,
     }
 
     @Test
-    public void testProjectGenerationFromEmptyPom() throws Exception {
+    public void testProjectGenerationWithExistingPomShouldFail() throws Exception {
         testDir = initProject("projects/simple-pom-it", "projects/project-generation-from-empty-pom");
         assertThat(testDir).isDirectory();
         invoker = initInvoker(testDir);
         InvocationResult result = setup(new Properties());
 
-        assertThat(result.getExitCode()).isZero();
-
-        assertThat(new File(testDir, "pom.xml")).isFile();
-        assertThat(new File(testDir, "src/main/java")).isDirectory();
-
-        assertThat(new File(testDir, "src/main/resources/application.properties")).exists();
-        assertThat(new File(testDir, "src/main/resources/META-INF/resources/index.html")).exists();
-
-        assertThat(FileUtils.readFileToString(new File(testDir, "pom.xml"), "UTF-8"))
-                .contains(getPluginArtifactId(), MojoUtils.TEMPLATE_PROPERTY_QUARKUS_PLUGIN_VERSION_VALUE,
-                        getPluginGroupId());
-
-        final Model model = loadPom(testDir);
-        assertThat(model.getProperties().getProperty(MojoUtils.TEMPLATE_PROPERTY_QUARKUS_PLUGIN_VERSION_NAME))
-                .isEqualTo(getPluginVersion());
-        assertThat(model.getProperties().getProperty(MojoUtils.TEMPLATE_PROPERTY_QUARKUS_PLATFORM_ARTIFACT_ID_NAME))
-                .isEqualTo(getBomArtifactId());
-
-        assertThat(model.getDependencyManagement().getDependencies().stream()
-                .anyMatch(d -> d.getArtifactId().equals(MojoUtils.TEMPLATE_PROPERTY_QUARKUS_PLATFORM_ARTIFACT_ID_VALUE)
-                        && d.getVersion().equals(MojoUtils.TEMPLATE_PROPERTY_QUARKUS_PLATFORM_VERSION_VALUE)
-                        && d.getScope().equals("import")
-                        && d.getType().equals("pom"))).isTrue();
-
-        assertThat(model.getDependencies()).isEmpty();
-
-        assertThat(model.getProfiles()).hasSize(1);
-        assertThat(model.getProfiles().get(0).getId()).isEqualTo("native");
```

Comment:
It looks like those asserts are important, why remove them?

Reply:
Migrating project has been removed from the "create" command, those asserts were for it. I am now testing that it fails. Note that the same asserts are already in the other "fromScratch" test.